src/app.js

```diff
@@ -1 +1,22 @@
 'use strict';
+
+const fs = require('fs');
+
+function funcRead() {
+  const terminal = process.argv.slice(2);
+  const filePath = './src/' + terminal[0];
+  const newFilePath = './src/' + terminal[1];
+  const isFile = fs.existsSync(filePath);
```

Comment:
You are not checking if the source and destination paths are the same. The task specifies that you should ensure that the file is not copied to the same location.